client/clientservice/src/main.cpp

```diff
@@ -51,6 +50,7 @@ po::variables_map parseCmdLine(int argc, char** argv) {
     ("config", po::value<std::string>()->required(), "YAML configuration file for the RequestService")
     ("host", po::value<std::string>()->default_value("0.0.0.0"), "Clientservice gRPC service host")
     ("port", po::value<int>()->default_value(50505), "Clientservice gRPC service port")
+    ("num-async-threads", po::value<int>()->default_value(1), "Number of threads for async gRPC services")
```

Comment:
No. The recommendation is to go with the number of CPUs.
The default of `1` still processes requests concurrently without issues in small setups. Only when we go on beefier infrastructure this will matter.

Reply:
Got it, thanks!